packages/@n8n/nodes-langchain/nodes/tools/ToolHttpRequest/ToolHttpRequest.node.ts

```diff
@@ -394,9 +394,14 @@ export class ToolHttpRequest implements INodeType {
 			optimizeResponse,
 		);
 
-		const description = prepareToolDescription(toolDescription, toolParameters);
+		const schema = makeToolInputSchema(toolParameters);
 
-		const tool = new DynamicTool({ name, description, func });
+		const tool = new N8nTool(this, {
+			name,
+			description: toolDescription,
+			func,
+			schema,
+		});
 
```

Comment:
are we sure that after update behavior would be same? if not I would suggest to put those changes under new minor version

Reply:
Do you mean the node version?

Reply:
@burivuhster 
yes, exactly